admin_ui/src/views/Home.vue

```diff
@@ -22,6 +22,9 @@ export default {
     mounted() {
         this.$store.commit("updateCurrentTablename", "")
     },
+    async created() {
+        await this.$store.dispatch("fetchMeta")
+    },
```

Comment:
It's also dispatching this in the create method of App.vue, so the API is being called twice. I wonder if the problem was just related to the fixes you made in `meta.ts`, and we don't need this additional API call? Does it still work for you locally without it?

Reply:
It's weird but works only if api call is from `Home.vue`. I completely remove api call from `App.vue` and  
leave api call from `Home.vue`, and everything work correctly.

Reply:
The only problem with just calling it in `Home.vue`, is if someone opens the website on a different page, it wouldn't call it. It will also get called every time `Home.vue` gets opened.

In theory, `App.vue` is the best place for it - as it should always get called, and only gets called once, but there could be a bug which is stopping it from working. Which browser do you use?

Reply:
Chromium Version 87.0.4280.66(64-bit) and Mozilla Firefox 84.0.2 (64-bit). My OS is Linux Mint 19.1. Is this working for you just with changes in `meta.ts`?

Reply:
I was able to replicate it when opening in an incognito window - but only some of the time. It's very strange. Probably means there's a race condition somewhere. I'll play around a bit more.

Reply:
I think I know what the issue is now. If you load the admin and you're not logged in, the app tries to fetch the meta, but if the user isn't logged in it fails. This is why adding it to the `Home.vue` fixed it.

Reply:
I'll add the API call to `Login.vue` instead - so when the user logs in, it fetches the meta. That should fix it!

Reply:
Well done! I tried to add an API call to `Login.vue` and everything works great.